README.md

```diff
@@ -61,3 +61,61 @@ One possible data model for this application is as follows:
 └──────────────────────────┘              └───────────────────────┘
 ```
 </details>
+
+
+## Specs
+<details>
+  <summary>Click here to see names of each test</summary>
+
+<li>The home page has an h1 element with the text "Your todo list"
```

Comment:
Are all of these list items missing closing tags? Also is the containing `<ul>` missing?

I believe you can use Markdown within the `<details>`, perhaps that may simplify?